containers/ecr-viewer/src/app/data/db/utils/migrate.ts

```diff
@@ -0,0 +1,114 @@
+import { promises as fs } from "fs";
+import path from "path";
+import { fileURLToPath } from "url";
+
+import { Kysely, Migrator } from "kysely";
+
+import { getDb, dbSchema } from "@/app/api/services/database";
+
+import { MultiDirectoryMigrationProvider } from "./multiDirectoryMigrationProvider";
+
+// Empty interface used only in migrations
+interface Database {}
+
+async function runMigration(
+  db: Kysely<Database>,
+  migrationsDir: string[],
+  command: string,
```

Comment:
```suggestion
  command: "up" | "down",
```
these are the only options, right?